### PR TITLE
SEO-2431-Updated-Retarget-keywords-FF-docs

### PR DESCRIPTION
**Pull request by Gowri-V-L**
Hi @Girija-Rajendran ,

Please find the retarget keywords in File formats, 
convert pdf to word - KB
edit pdf in word - FT
pdf forms - FT, UG
pdf invoices - FT, UG
pdf to qr code - not valid so worked for QR code in KB
view pdf - FT
how to convert pdf to excel without software - KB
pdf generation - UG
excel protected view - FT, UG
excel spreadsheet templates - UG, FT

FT master:https://github.com/syncfusion-content/featuretour-file-formats/pull/203
FT dev:https://github.com/syncfusion-content/featuretour-file-formats/pull/202

### CODE REVIEW

File: File-Formats/XlsIO/Working-with-Template-Markers.md [hunk 1]
```diff
@@ -497,7 +497,7 @@ By default, DataTable values will be filled in the worksheet as a string format.
 
  The following screenshot represents the input template which has a template marker.
 
-![Bind from DataTable Example](Working-with-Template-Markers_images/Working-with-Template_Markers_img3.jpeg)
+![Bind from DataTable Example](Working-with-Template-Markers_images/Bind_datatable_to_Marker_in_Excel_Spreadsheet_Templates.jpeg)
```
Comment (Girija-Rajendran) [inline File-Formats/XlsIO/Working-with-Template-Markers.md:500]: Please check with product team regarding this
image

File: File-Formats/PDF/Create-PDF-file-in-ASP-NET-Core.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ A complete work sample can be downloaded from [Create-PDF-file.zip](http://www.s
 
 
 By executing the program, you will get the PDF document as follows.
-![Generate PDF using .NET Core](GettingStarted_images/Hello_World.jpg)
+![PDF Generation output](GettingStarted_images/pdf-generation-output.jpg)
```
Comment (Girija-Rajendran) [inline File-Formats/PDF/Create-PDF-file-in-ASP-NET-Core.md:94]: add platform name in alt text as well as image name

File: File-Formats/XlsIO/Working-with-Template-Markers.md (hunk 1 above)
Comment (Gowri-V-L) [reply File-Formats/XlsIO/Working-with-Template-Markers.md]: confirmed with product team , the current change was fine
image

File: File-Formats/PDF/Create-PDF-file-in-ASP-NET-Core.md (hunk 2 above)
Comment (Gowri-V-L) [reply File-Formats/PDF/Create-PDF-file-in-ASP-NET-Core.md]: added in alt name, but due to common image reference, didn't changed in image name

File: File-Formats/XlsIO/Working-with-Template-Markers.md [hunk 3]
```diff
@@ -497,7 +497,7 @@ By default, DataTable values will be filled in the worksheet as a string format.
 
  The following screenshot represents the input template which has a template marker.
 
-![Bind from DataTable Example](Working-with-Template-Markers_images/Working-with-Template_Markers_img3.jpeg)
+![Bind from DataTable Example](Working-with-Template-Markers_images/Bind_data_from_datatable_to_Excel_Spreadsheet_Template.jpeg)ss
```
Comment (Girija-Rajendran) [inline File-Formats/XlsIO/Working-with-Template-Markers.md:500]: remove the unwanted text
image

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**